cmd/syncRules.go

```diff
@@ -163,7 +238,20 @@ func NewSyncRulesCommand() *cobra.Command {
 						Severity:     rego.Severity,
 						RuleText:     rego.Text,
 					}
-					client.CustomRules.CreateCustomRule(params, auth)
+
+					created, err := client.CustomRules.CreateCustomRule(params, auth)
+					if err != nil {
+						log.Fatal(err)
+					}
+
+					if ruleFamilies != nil {
+						famParams := custom_rules.NewUpdateCustomRuleParams()
+						famParams.RuleID = created.Payload.ID
+						famParams.Rule = &models.UpdateCustomRuleInput{
+							Families: ruleFamilies,
+						}
+						client.CustomRules.UpdateCustomRule(famParams, auth)
```

Comment:
Probably needs an extra `err` check here.

Reply:
I don't mind adding it (that would actually be my personal preference as well), but it seemed like *not* checking the err here was the norm.  For example, in he original code there wasn't a check on either call in this file (for example, see line 266 below), and I don't think other commands were checked either.